omnigibson/object_states/toggle.py

```diff
@@ -89,7 +89,10 @@ def overlap_callback(hit):
         def check_overlap():
             nonlocal valid_hit
             valid_hit = False
-            og.sim.psqi.overlap_mesh(*projection_mesh_ids, reportFn=overlap_callback)
+            if self.visual_marker.prim.GetTypeName() == "Mesh":
```

Comment:
these should all hopefully be standardized by 2023.1.0. I forgot which one is no longer needed but we should only stick with one type